src/headless.cpp

```diff
@@ -34,9 +34,7 @@ int main(int argc, char *argv[])
     }
 
     uint64_t num_steps = std::stoul(argv[2]);
-    std::vector<std::string> scenes = {"../data/processed/examples/tfrecord-00001-of-01000_307.json",
-         "../data/processed/examples/tfrecord-00003-of-01000_109.json",
-         "../data/processed/examples/tfrecord-00012-of-01000_389.json"};
+    std::vector<std::string> scenes = {"/home/nadarenator/Desktop/Projects/gpudrive/data/processed/validation/tfrecord-00005-of-00150_3.json"};
```

Comment:
@nadarenator hardcoded link that I think should be removed?

Reply:
yess will do

Reply:
fixed